### PR TITLE
math header refix

### PR DESCRIPTION
**Pull request by its-pointless**
patch c++ marh header

### CODE REVIEW

File: packages/ndk-sysroot/math-header.diff [hunk 1]
```diff
@@ -0,0 +1,22 @@
+--- data/data/com.termux/files/usr/include/c++/v1/cmath.orig	2019-03-24 00:31:49.512145195 +0000
++++ ./data/data/com.termux/files/usr/include/c++/v1/cmath	2019-03-24 00:32:19.392703442 +0000
+@@ -302,7 +302,7 @@
+ */
+ 
+ #include <__config>
+-#include <math.h>
++#include <c++/v1/math.h>
```
Comment (tomty89) [inline packages/ndk-sysroot/math-header.diff]: Why not `#include "math.h"`?

**Comment by tomty89** [comment]
Shouldn't this be put under `ndk-patches` as a `.patch`? (and bump the version in `TERMUX_STANDALONE_TOOLCHAIN`)

File: packages/ndk-sysroot/math-header.diff (hunk 1 above)
Comment (its-pointless) [reply packages/ndk-sysroot/math-header.diff]: Because it was done this way last time....

**Comment by its-pointless** [comment]
Not neccesary since the bug doesn't manifest when using ndk.

**Comment by tomty89** [comment]
Also you should avoid hardcoding a TERMUX_PREFIX in the diff. `cd` (in a group) instead.

**Comment by buttaface** [comment]
Any reason this isn't in? I had to apply this patch manually to the sysroot headers when building Swift in Termux, apple/swift#23208.

**Comment by buttaface** [comment]
Thanks, looking forward to not having to re-patch this manually on every sysroot update. :wink:

**Pull request merged**